plugins/modules/zos_started_task.py

```diff
@@ -0,0 +1,1639 @@
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from __future__ import (absolute_import, division, print_function)
+
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+module: zos_started_task
+version_added: 1.16.0
+author:
+  - "Ravella Surendra Babu (@surendrababuravella)"
+short_description: Perform operations on started tasks.
+description:
+  - start, display, modify, cancel, force and stop a started task
+
+options:
+  arm:
+    description:
+        - I(arm) indicates to execute normal task termination routines without causing address space destruction.
+        - Only applicable when I(state) is C(forced), otherwise ignored.
+    required: false
+    type: bool
+  armrestart:
+    description:
+        - Indicates that the batch job or started task should be automatically restarted after CANCEL or FORCE
+          completes, if it is registered as an element of the automatic restart manager. If the job or task is
+          not registered or if you do not specify this parameter, MVS will not automatically restart the job or task.
+        - Only applicable when I(state) is C(cancelled) or C(forced), otherwise ignored.
+    required: false
+    type: bool
+  asidx:
+    description:
+        - When I(state) is C(cancelled), C(stopped) or C(forced), I(asidx) is the hexadecimal address space
+          identifier of the work unit you want to cancel, stop or force.
+        - Only applicable when I(state) is C(stopped), C(cancelled), or C(forced), otherwise ignored.
+    required: false
+    type: str
+#   device_type:
+#     description:
+#         - Type of the output device (if any) associated with the task.
+#         - Only applicable when I(state) is C(started), otherwise ignored.
+#     required: false
+#     type: str
+#   device_number:
+#     description:
+#         - Number of the device to be started. A device number is 3 or 4 hexadecimal digits. A slash (/) must
+#           precede a 4-digit number but not a 3-digit number.
+#         - Only applicable when I(state) is C(started), otherwise ignored.
+#     required: false
+#     type: str
+  dump:
+    description:
+        - Whether to perform a dump. The type of dump (SYSABEND, SYSUDUMP, or SYSMDUMP)
+          depends on the JCL for the job.
+        - Only applicable when I(state) is C(cancelled), otherwise ignored.
+    required: false
+    type: bool
+  identifier_name:
+    description:
+        - Option I(identifier_name) is the name that identifies the task. This name can be up to 8
+          characters long. The first character must be alphabetical.
+    required: false
+    type: str
+    aliases:
+        - identifier
+  get_system_logs:
```

Comment:
Let's change this option's name to `system_logs`, we are trying to avoid the verb options for the new 2.0 version so is better to have the same guidelines applied here.